test/e2e/storage/volumes.go

```diff
@@ -503,24 +503,21 @@ var _ = utils.SIGDescribe("Volumes", func() {
 	Describe("vsphere [Feature:Volumes]", func() {
 		It("should be mountable", func() {
 			framework.SkipUnlessProviderIs("vsphere")
+			vspheretest.Bootstrap(f)
+			nodeList := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
+			Expect(nodeList.Items).NotTo(BeEmpty(), "Unable to find ready and schedulable Node")
+			nodeInfo := vspheretest.TestContext.NodeMapper.GetNodeInfo(nodeList.Items[0].Name)
```

Comment:
Aren't you only getting the first node's Name here? If so then maybe declaring _nodeName_ is more meaningful than nodeInfo?